### PR TITLE
insta d team

### PR DESCRIPTION
**Pull request by cosmicboon**
9월27일 데일리 @JeffGuKang

### CODE REVIEW

File: daily/01_Instagram/README.md [hunk 1]
```diff
@@ -233,14 +233,28 @@
 	- es6 의 차이점과 promise 개념
 	- Webpack과 babel 의 관계 이해
 
+<<<<<<< HEAD
```
Comment (mu-hun) [inline daily/01_Instagram/README.md]: Menual merge isn't done yet!

수동 머지가 잘 안되었습니다.

File: daily/01_Instagram/README.md [hunk 2]
```diff
@@ -251,3 +265,25 @@
 		- karma로 테스트 실행(npm run test)
 		- FireFox, 크롬 미설치시 버그 발견, 이슈 작성
 		- 오프라인 모임!!(사진 개인적으로 전송 부탁드립니다.^^)
+>>>>>>> eda9977f4743e8ad307025b94800f9dcd0c7feb2
```
Comment (MuhunKim) [inline daily/01_Instagram/README.md]: .

File: daily/01_Instagram/README.md [hunk 3]
```diff
@@ -233,14 +233,28 @@
...
+
+  - Team Goal
+  	- Webpack 이해하고 사용하기
+  	- Git 숙달
+  	- ES6 문법에 맞춘 코드 스타일 통일
+
+  - 박경원
+    - pr을 위한 프로세스 이해 및 실습
+    - 히스토리 보며 코드 비교 학습
+=======
```
Comment (MuhunKim) [inline daily/01_Instagram/README.md]: .

**Review by mu-hun — CHANGES_REQUESTED** [review]

File: daily/01_Instagram/README.md (hunk 2 above)
Comment (JeffGuKang) [inline daily/01_Instagram/README.md]: Delete unnecessary traces.

File: daily/01_Instagram/README.md [hunk 4]
```diff
@@ -251,3 +265,25 @@
...
+
+#### Task
+
+- Team Goal
+	- Webpack 이해하고 사용하기
+	- Git 숙달
+	- ES6 문법에 맞춘 코드 스타일 통일
+
+- 신미르
+  - Git 정리
```
Comment (JeffGuKang) [inline daily/01_Instagram/README.md:315]: Use `tab` for the nested list.

File: daily/01_Instagram/README.md [hunk 5]
```diff
@@ -233,14 +233,28 @@
 	- es6 의 차이점과 promise 개념
 	- Webpack과 babel 의 관계 이해
 
+<<<<<<< HEAD
+  ## 2016년 9월 27일
```
Comment (JeffGuKang) [inline daily/01_Instagram/README.md]: Do not use unnecessary spaces.

File: daily/01_Instagram/README.md [hunk 6]
```diff
@@ -233,14 +233,28 @@
...
+
+  #### Task
+
+  - Team Goal
+  	- Webpack 이해하고 사용하기
+  	- Git 숙달
+  	- ES6 문법에 맞춘 코드 스타일 통일
+
+  - 박경원
+    - pr을 위한 프로세스 이해 및 실습
```
Comment (JeffGuKang) [inline daily/01_Instagram/README.md]: Use `tab`.

**Review by JeffGuKang — CHANGES_REQUESTED** [review]
Do not use unnecessary spaces. 
Use `tab` for nested lists.

**Pull request closed without merging**